### ISSUE TITLE
fails to create makefile if username contains .c

### ISSUE THREAD

**Issue by ankitchachada** [pganalyze/pg_query#202]
I was trying to install the gem on my machine and my username(on my mac machine) is ankit.chachada but due to https://github.com/pganalyze/pg_query/blob/024a013924ce9d0ff3d6fa327f25274ad5934c3c/ext/pg_query/extconf.rb#L8 while executing extconf.rb it is replacing **.c to .o** which is failing to install the gem.
Following is the output:
```
current directory: /Users/ankit.chachada/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/pg_query-2.0.1/ext/pg_query
make "DESTDIR="
linking shared-object pg_query/pg_query.bundle
clang: error: no such file or directory: '/Users/ankit.ohachada/
```

I did a dirty fix to resolve this by adding new line`$objs = Dir.glob(File.join(__dir__, '*.oh')).map { |f| Pathname.new(f).sub_ext('.c').to_s }` will see if I can patch it up. Thanks

**Comment by lfittl** [comment]
@ankitchachada Thanks for the report - this was already fixed in https://github.com/pganalyze/pg_query/pull/198 - I'll cut a new release today with the fix.

**Comment by lfittl** [comment]
@ankitchachada Released now as 2.0.2 :)
